Add a list of forks with prebuilt servers for player convenience (#313)
space-wizards/docs: src/en/server-hosting/popular-forks-cdns.md

| Harmony 14 | [Link](http://cdn.harmony14.com/fork/harmony/) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
| Solaris 14 | [Link](https://ss14-cdn.mw-gc.com/fork/MWGSolaris14) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
| Axolotl MRP | [Link](https://axolotl.yuniiworks.de/cdn/fork/axolotl/) | Linux (x64) |

FAQ:

- The fork I want is not listed here.
	- You will need to contact the fork you want a server build for and ask them if they provide one. If they do suggest them to add themselves here.
	
- The fork I want is listed here, but my platform is not available for download.
	- You will need to contact the fork asking if they are willing to build for your platform.
	
- The fork does not provide prebuilds (or any other issue where you are unable to retrive a server prebuild)
	- Your only choice is to compile the forks codebase yourself if available. Instructions can be found [here](../general-development/setup/server-hosting-tutorial.md#level-2-server-with-custom-code)

- I'm a server host and I want to add myself here.
	- You are free to make a PR yourself in the docs or ping @vasilisiscool (322708417540259841) in #hosting on discord with all the information needed.

> <!-- 

Reviewer: I think this should be visible and not in a comment?

Author: Thats not a comment

Reviewer: It's not visible when rendered: https://github.com/space-wizards/docs/blob/892c9b9de6aae1649447a6f520086dcf37a4db71/src/en/server-hosting/popular-forks-cdns.md

Author: It is visible in mdbook

Reviewer: I meant the rules, not the FAQ. The `<!--` comments it out so it doesn't render.

Author: I was confused since you reviewed a completely unrelated section. Most players don't care, so I commented it out. It's mostly for the reviewing maintainer. I guess its fine to include now that i think about it

Reviewer: I reviewed the line that started the comment, sorry if it was unclear.

Rules for listing:

> 1. Must have a decent playerbase on the hub and been on for at least a week.

Reviewer: Should probably clearly define "decent playerbase" since people may have different opinions on what's considered a decent playerbase.

Reviewer 2: Ours (Solaris 14) probably doesn't even meet that out of the gate. We have roughly five regulars and are still working on our own content to distinguish our fork from Wizden. I wouldn't pitch a fit if we were removed, but think it is awesome that we are on the list :)

2. Must be a simple link to the download source, no discord invites or anything like that.
3. Must have at least one server platform.
4. No protected forks (Require a password for access to server builds)
5. 18+/ERP forks must be expliticly stated in the fork name.
6. Dehubbed servers or any servers breaking these rules will be removed.

Template

> | SERVER NAME | [Link](SERVER_LINK) | Windows, macOS (x64/Intel), Linux (x64/ARM64) | 

Reviewer: Could there be a column for the software license(s) that the fork is under? I think that's pretty important to know if you're hosting, considering some forks use AGPL.

Author: In most cases, users who download prebuilds are only interested in a sandbox environment. But i will add it

# List of codebases with prebuilt server packages

This document will serve as a list of forks known to have a prebuilt server software for players to download.

> This doc was last updated at 2024-10-01 13:52:04 UTC

Reviewer 3: Probably don't need to specify the time of day too.

Author: PRECISION

```admonish danger

> All forks listed here (Exept Wizard's Den) are **not vetted by Wizard's Den**. We only just put the links in a convinient place.

Reviewer 4: ```suggestion
All forks' builds listed here (Except Wizard's Den) are **not vetted by the Space Wizards**. We only put the links here for convenience.
```

Reviewer: Maybe it should say "not vetted by Space Wizards **management**" since Space Wizards includes anyone including contributors.

> Third party server builds are not affiliated with the project, and have the samse risks as running any random executable off the internet. Only run them if you trust them to not take over your computer

Reviewer: ```suggestion
Third party server builds are not affiliated with the project, and have the same risks as running any random executable off the internet. Only run them if you trust them to not take over your computer.
```

I would remove the last sentence, or maybe change it to just "Only run software that you trust." There are other bad things that malicious software can do that isn't "take over your computer."

Author: Oops someone else wrote this for me and i did not notice this
